bricks/data-view/src/title-text/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { CSSProperties } from "react";
+import { createDecorators } from "@next-core/element";
+import { ReactNextElement } from "@next-core/react-element";
+import variablesStyleText from "../data-view-variables.shadow.css";
+import styleText from "./styles.shadow.css";
+
+const { defineElement, property } = createDecorators();
+type TitleType = "normal" | "stroke" | "gradient";
+interface TitleTextProps {
+  text: string;
+  type?: TitleType;
+  fontSize?: CSSProperties["fontSize"];
+  fontWeight?: CSSProperties["fontWeight"];
+  letterSpacing?: CSSProperties["letterSpacing"];
+}
+
+/**
+ * @id data-view.title-text
+ * @name data-view.title-text
+ * @docKind brick
+ * @description 大屏标题文本构件
+ * @author astrid
+ * @noInheritDoc
```

Comment:
注意看下 contributing guide 里的[关于注释的说明](https://github.com/easyops-cn/next-bricks/blob/master/CONTRIBUTING.md#%E6%BA%90%E4%BB%A3%E7%A0%81%E6%B3%A8%E9%87%8A)

```suggestion
 * 大屏标题文本构件
 * @author astrid
```